learn/getting_started/installation.md

```diff
@@ -85,7 +85,14 @@ git clone https://github.com/meilisearch/MeiliSearch
 cd MeiliSearch
 ```
 
-In the cloned repository, compile MeiliSearch.
+In the cloned repository, you can choose which release you want to use.
+See the list of all release [here](https://github.com/meilisearch/MeiliSearch/releases).
+Then run the following command whith whichever tags you selected:
+```bash
+git checkout v0.25.2
```

Comment:
It would be cool if we could update the tag automatically to the latest version here but I don't know if it's possible.

Reply:
You can set symbolic tags and reset them when you want to change the commit to which they point.  Then you have tag `v0.25.2` and tag `release` pointing to the same commit and users can say "I want to be at `release` without worrying about which that is.

That's especially helpful in automated environments, where I might have a script that automatically looks for a well-known tag.

Reply:
Hey @JeffAbrahamson, I was thinking about that, and I was wondering, isn't it way more straightforward and less confusing for our user to get a link to the latest GitHub release: https://github.com/meilisearch/meiliSearch/releases/latest
And then, the user can download a simple tarball without even using git or a terminal.

What do you think?

Reply:
The tarball contain the source code.
To me people fluent with git will have no problem with the link I gave since you can see which tag is associated with the release.
And people who don’t want to clone a whole repo because only want this version will be able to download only what they need in a compressed manner without the burden of knowing how to use git.

To me this is a win-win situation.
And at least we won’t ever give the impression that you should just use latest commit on `main` which was the initial issue.